docs/sources/contribute/release-notes/index.md

```diff
@@ -34,185 +34,102 @@ The Grafana Cloud Traces updates are published using the What's new.
-       </tr>
-       <tr>
-         <th scope="row">DOCUMENTATION URL (OPTIONAL)</th>
-         <td>URL to the public documentation for this feature.</td>
-         <td></td>
-       </tr>
-       <tr>
-         <th scope="row">ENABLEMENT VIDEO (OPTIONAL)</th>
-         <td>Link to the video used for enablement.</td>
-         <td>
-           <p>Enablement videos are perhaps the fastest, most engaging tool for employees and users to learn about your feature. Use them for maximum engagement.</p>
-           <p>Follow these instructions to create and upload a video: <a href="https://docs.google.com/document/d/1nCiG62FxJ9J_qLTnlzNopSsT-VEAlxCUCsrAEpWL68U/edit#heading=h.fierz9i4q8ft">Enablement video instructions</a>.</p>
-           <p>When you upload an enablement video, the Content team receives a notification, edits it for the public, and uploads it to YouTube to coincide with your feature's release. They need a few weeks' lead time for this.</p>
-         </td>
-       </tr>
-       <tr>
-         <th scope="row">INTERNAL INFORMATION (OPTIONAL)</th>
-         <td>Information for Grafana Labs employees only.</td>
-         <td>
-           <p>For example, ProductDNA, Slack channel, FAQ, training documentation, or videos.</p>
-           <p>Used for training and internal announcements.</p>
-           <p>This is only visible on the <a href="https://admin.grafana.com/whats-new/">internal What's new page</a>, <em>not</em> the <a href="https://grafana.com/docs/grafana-cloud/whats-new/">public What's New page</a>.</p>
-         </td>
-       </tr>
-     </tbody>
-   </table>
+When you're ready to add a What's new entry, complete the following steps:
 
+1. Go to the What's new collection of the [website CMS](https://admin.grafana.com/content-admin/#/collections/whats-new) and click **New Feature** at the top of the page.
+1. Fill out [the CMS fields](#cms-fields).
 1. Click **Save**.
    The entry is now in **Draft** status and the CMS opens a pull request in the `grafana/website` repository.
 
-   If your entry is future-dated, it won't show up in the website-generated preview or in your local build, but you can see it in the preview of the internal feed.
-   To do so, remove `/docs/grafana-cloud` from the preview URL and add the heading for your entry at the end of the URL.
-   It should look something like this: `https://deploy-preview-18347-zb444pucvq-uw.a.run.app/whats-new/#create-subtables-in-table-visualizations`.
+1. Preview your entry in context.
+
+- **Release date in the past** - Your entry will be visible in the website-generated preview.
+
+- **Release date in the future** - Your entry is only visible in the preview of the _internal feed_.
+  To see it, in the deploy preview URL, replace `/docs/grafana-cloud` with the heading for your entry.
+  For example: `https://deploy-preview-18347-zb444pucvq-uw.a.run.app/whats-new/#create-subtables-in-table-visualizations`.
 
 1. If your entry is ready to publish, proceed to the next step.
```

Comment:
Does this numbering start over because the bulleted list above is left-aligned?

Reply:
Yep, good catch. I'll fix that and push to the PR again.